src/classes/MAPR_CON_PreferredEmailFields.cls

```diff
@@ -0,0 +1,254 @@
+    }
+
+    /**************************************************************************************
+    * @description Returns a map of Contact Email Field Describes by their API Names
+    * @return Map<String, Schema.DescribeFieldResult>
+    **************************************************************************************/
+    @TestVisible
+    private Map<String, Schema.DescribeFieldResult> getContactEmailFieldByApiName() {
+        return UTIL_Describe.getFieldsOfType('Contact', 'EMAIL');
+    }
+
+    /**************************************************************************************
+    * @description Returns a map of Preferred Email's picklist values by their labels
+    * @return Map<String, String>
+    **************************************************************************************/
+    @TestVisible
+    private Map<String, String> getPreferredEmailPicklistValuesByLabel() {
+        return UTIL_Describe.getpicklistEntriesLabelToAPIMap('Contact', UTIL_Namespace.StrTokenNSPrefix('') + 'Preferred_Email__c');
+    }
+
+    /**************************************************************************************
+    * @description Returns a map of Preferred Email's Legacy picklist values by their labels
+    * @return Map<String, String>
+    **************************************************************************************/
+    @TestVisible
+    private Map<String, String> getLegacyPreferredEmailPicklistValuesByLabel() {
+
+        Map<String,String> legacyPreferredEmailPicklistValuesByLabel = new Map<String,String> {
+                  'University Email' => 'University',
+                 'Work Email' => 'Work',
+                 'Alternate Email' => 'Alternate'
+                };
+      
+        return legacyPreferredEmailPicklistValuesByLabel;            
+    }
+
+    /**************************************************************************************
+    * @description Returns a map of Preferred Email's picklist Labels by their API names
+    * @return Map<String, String>
+    **************************************************************************************/
+    @TestVisible
+    private Map<String, String> getPreferredEmailPicklistLabelByApiName() {
+        return UTIL_Describe.getPicklistEntriesApiToLabelMap('Contact', UTIL_Namespace.StrTokenNSPrefix('') + 'Preferred_Email__c');
+    }
+
+    /**************************************************************************************
+    * @description Returns True if the API field name passed is an EDA field else false
+    * @param fieldApiName The Contact Field's API name to check for
+    * @return Boolean
+    **************************************************************************************/
+    @TestVisible
+    private Boolean isEDAField(String fieldApiName) {
+
+        String namespace = UTIL_Namespace.getNamespace();
+
+        if (Test.isRunningTest() && String.isBlank(namespace) && fieldApiName.endsWithIgnoreCase('__c')) {
```

Comment:
@MatthewBlanski wanted to check with you if this was ok to make tests pass in non-namespaced orgs.